src/openzaak/components/autorisaties/admin_views.py

```diff
@@ -68,7 +63,7 @@ def is_local_url(autorisatie):
 def get_initial_for_component(
     component: str,
     autorisaties: List[Autorisatie],
-    spec: Optional[AutorisatieSpec] = None,
+    catalogus_autorisaties: Optional[List[CatalogusAutorisatie]] = None,
```

Comment:
1. I tested it and currently it saves only one CatalogusAutorisatie with the scopes of the last row. I think we should either raise an error here or make scopes part of `unique_together` to allow multiple CatalogusAutorisaties with same component/catalogus but different scopes
2. if the scopes are the same, you'll get an error stating that there are overlapping scopes, if not the same behaviour as above (you'll end up with one CatalogusAutorisatie and the confidentiality from the last row is set on that)
3. In this case, both are created. I think (like I mentioned in the other comment) we should probably raise errors if there is this kind of overlap

I'll add validation for this :+1:

Reply:
Upon second thought, I'm not sure if we have to raise an error in the 3rd example, currently this works for regular autorisaties as well (if the scopes have no overlap)

Reply:
I agree with you! Let's raise an error when there is a potential overlap, because it's better than having questions from clients later who have overlapping CatalogusAutorisatie and single Autorisatiesand who don't understand why the authorization has worked a particular way

Reply:
checked it, this error is raised if the scopes are the same (regardless of confidentiality) 
![image](https://github.com/user-attachments/assets/ed2b5535-3333-49d7-97de-e8d39ae96790)